pkg/sql/testutils.go

```diff
@@ -126,3 +129,36 @@ func (dsp *DistSQLPlanner) Exec(
 	dsp.PlanAndRun(ctx, evalCtx, planCtx, p.txn, p.curPlan.plan, recv)()
 	return rw.Err()
 }
+
+// QueryExpect runs the query string using the gosql.DB and compares to queries
+// row values with the expected values.
+// expected: a slice of slices of type interface{} containing the row values.
+// 		Example:
+//			expected := [][]interface{}{{1, "1", 1}, {2, "2", 2}, {3, "HELLO", 3}}
+// dest: pointers to the desired types for the Scan to read into.
+//			for the example above, dest should contain *int, *string, *int.
+func QueryExpect(
```

Comment:
we must have something that does this already? Either way, i think wherever this is used a combination of the normal query + require.Equal() would suffice.